ts/src/cryptocom.ts

```diff
@@ -759,40 +760,39 @@ export default class cryptocom extends Exchange {
             'instrument_name': market['id'],
         };
         if (since !== undefined) {
```

Comment:
@Dan-krm for some reason, `since` does not seem to be working, can you test it on your end?
```
✗ n cryptocom fetchTrades "BTC/USDT" 1686927835000 3          
Debugger attached.
2023-06-17T16:07:20.148Z
Node.js: v18.14.0
CCXT v3.1.41
cryptocom.fetchTrades (BTC/USDT, 1686927835000, 3)
2023-06-17T16:07:22.467Z iteration 0 passed in 389 ms

                 id |     timestamp |                 datetime |   symbol | order | side | takerOrMaker |    price |  amount |         cost | type | fee | fees
---------------------------------------------------------------------------------------------------------------------------------------------------------------
4611686018447577596 | 1687017854605 | 2023-06-17T16:04:14.605Z | BTC/USDT |       |  buy |              | 26394.99 | 0.04999 | 1319.4855501 |      |  {} | [{}]
4611686018447577597 | 1687017854607 | 2023-06-17T16:04:14.607Z | BTC/USDT |       |  buy |              | 26394.99 | 0.01999 |  527.6358501 |      |  {} | [{}]
4611686018447577598 | 1687017860284 | 2023-06-17T16:04:20.284Z | BTC/USDT |       |  buy |              | 26395.19 | 0.00491 |  129.6003829 |      |  {} | [{}]
```